File: codekata/basics/composite_number.py
# print 'yes' if it is composite number else print 'no'
# get num input from user
num=int(input())
# define composite function
def composite():
# initialize c is equal to zero
  c=0
  for i in range(1, num + 1):
    if num % i == 0:
      d=i
      c=c+1
# if condition checks c is greater than 2
  if(c>2):
    print("yes")
  else:
    print("no")
composite()
        

File: codekata/javascript/basics/composite_number.js
// Given a number N, print 'yes' if it is composite else print 'no'
const readline = require('readline');
const inp = readline.createInterface({
  input: process.stdin
});
inp.on("line", (data) => {
var c=0
  for (i=1;i<=data+1;i++)
  {
    if (data % i == 0)
    {
     var d=i
      c=c+1
    }
  }
  if(c>2)
    console.log("yes");
  else
    console.log("no");
});  
        
